tests/console/zypper_extend.pm

```diff
@@ -104,7 +104,7 @@ sub run {
     zypper_call 'renamerepo "packman" Packman';
 
     #Remove a repository
-    zypper_call 'rr packman';
+    zypper_call 'rr Packman';
```

Comment:
Did this previously fail or is this just a noop typo fix?

Reply:
https://openqa.opensuse.org/tests/1300214#step/zypper_extend/71
the zypper_extend PR contained an error - the repo is added as packman, then renamed to Packman and finally packman (old name) being attempted to be removed; this fix does not strictly belong to this PR

Reply:
```
# zypper -n renamerepo "packman" Packman | cat; ( exit ${PIPESTATUS[0]} ); echo hdD~X-$?-
Repository 'packman' renamed to 'Packman'.
hdD~X-0-
# zypper -n rr packman | cat; ( exit ${PIPESTATUS[0]} ); echo xBCwK-$?-
Repository 'packman' not found by alias, number or URI.
xBCwK-0-
```
I assume it was just an unintentional left over.